src/style/colors.ts

```diff
@@ -131,6 +131,15 @@ export class Colors {
     return ColorName.name(color)[1];
   }
 
+  getSystemColorByHex(colorsOptions: string[], colorValue: string) {
```

Comment:
I think that the first argument of this function should be the `colorValue`, and the second argument can be optional (if the user doesn't pass the white list, all the colors are valid)
And also consider changing the `colorsOptions` name to something more meaningful, `validColors` for example.
WDYT?